README.md

```diff
@@ -1,2 +1,61 @@
 # issue-sync
-A tool for synchronizing issues between GitHub and JIRA
+
+Issue-sync is an automated tool for synchronizing GitHub and JIRA issues.
+It grew out of a desire to maintain a public GitHub repo while tracking
+private issues in a JIRA board; rather than require people to keep up
+with both sources, we decided to make *one* the single source of truth.
+
+## Usage
+
+To use, first ensure you have a JIRA server with the project you want
+to track on it - it can be a cloud account, or self-hosted. As well,
+make sure you have a user account which can access the project and
+create issues on it; it's recommended that you create an account
+specifically for the issue-sync tool.
+
+To the project, add the following custom fields: `GitHub ID`, `GitHub
```

Comment:
Simplify: `Add the following custom fields to the project`. Inverting the sentence and separating it into to clauses is extra confusing